DatadogSDKBridge/Classes/Implementation/GlobalState.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2021 Datadog, Inc.
+ */
+
+import Foundation
+
+internal struct GlobalState {
+    /// Holds global attributes to be used with Logs and Traces, because they don't expose Global instances, unlike RUM
+    internal static var globalAttributes: [String: Encodable] = [:]
+
+    internal static func addAttribute(forKey: String, value: Encodable?) {
+        if value == nil {
+            GlobalState.globalAttributes.removeValue(forKey: forKey)
+        } else {
+            GlobalState.globalAttributes[forKey] = value
+        }
```

Comment:
You can simplify to:
```suggestion
        GlobalState.globalAttributes[forKey] = value
```

Reply:
thanks, implemented